test/metabase/test/util/log.clj

```diff
@@ -195,9 +202,11 @@
         logger        (exact-ns-logger a-namespace)]
     (try
       (.addAppender logger appender (->Level level) nil)
+      ;; prevent events to be passed to the parent logger's appenders
+      ;; https://logging.apache.org/log4j/2.x/manual/configuration.html#Additivity
       (f (fn [] (appender-logs appender)))
       (finally
-        (.removeAppender logger appender-name)))))
+       (.removeAppender logger appender-name)))))
```

Comment:
In my setup the original indentation is followed.

Reply:
Yes, `finally` bodies should be indented 2 spaces.

https://guide.clojure.style/#body-indentation

```suggestion
        (.removeAppender logger appender-name)))))
```